pkg/inspector/image-inspector.go

```diff
@@ -185,7 +187,12 @@ func (i *defaultImageInspector) Inspect() error {
 	}
 
 	if i.imageServer != nil {
-		return i.imageServer.ServeImage(&i.meta, scanResults, scanReport, htmlScanReport)
+		handler, err := i.imageServer.GetHandler(&i.meta, scanResults, scanReport, htmlScanReport)
+		if err != nil {
+			return fmt.Errorf("failed to initialize imageserver: %v", err)
+		}
+		log.Printf("Serving image content %s on webdav://%s%s", i.opts.DstPath, i.opts.Serve, CONTENT_URL_PREFIX)
+		return http.ListenAndServe(i.opts.Serve, handler)
```

Comment:
unfortunately that makes testing Image server a lot harder (the idiomatic way to test http endpoints is using `httptest` which requires an `http.Handler`)